### DISCUSSION TITLE
Dropbox sync

### DISCUSSION THREAD

**Discussion by pietrop** [pietrop/digital-paper-edit-electron#72] [Ideas]
**Is your feature request related to a problem? Please describe.**

From @alvinschang 💡 

Backing up DPE data, as well as working across computers. Eg two use stories

> as users I want piece of mind that my data is being backed up automatically 

> as a user I want to be able to work across my work and personal computer on the same project

**Describe the solution you'd like**

In settings panel, have an advanced option, for pro users, to change the location of their data to a folder of their choosing, eg dropbox.

under the hood when you change folder and hit confirm, it would copy the content of the folder in the user library to the new folder.

it would leave/introduce a new a json in the default, library user app data folder, with the path of the custom data folder. eg If not present uses default, if present, uses custom. (Might need some notes in docs to document the implementation )

The average user might not use this as it could leave to issues and unexpected behaviour.

**Describe alternatives you've considered**

- not implementing this
- firebase web app (wouldn't work with larger file)

**Additional context**

This could lend itself to multi user collaboration, but is a suboptimal solution for that, a firebase web app would be better suited to handle that kind of use cas

**Comment by nigelhaslam** [comment]
I would love this feature.  It would enable me to work remotely with clients performing paper edits for me to polish and finish.
